### PR TITLE
TW-1534: refactor message item

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/217696678-56b19e88-fbf1-4564-9dbf-668c9f9b2a1e.mov

### CODE REVIEW

**Comment by dab246** [comment]
> Screen.Recording.2023-02-07.at.11.19.20.mov

Unrelated but I see the message doesn't scroll to the correct location where the last message was sent. We should improve it @hoangdat

**Comment by hoangdat** [comment]
@ManhNTX please add demo video also for Android

**Comment by hoangdat** [comment]
Please re-check the color and size of divider

**Review by hoangdat — COMMENTED** [review]
please dont forget to reduce the `DROPDOWN_SEPARATOR_HEIGHT`

File: lib/widgets/message/emoji_set.dart [hunk 1]
```diff
@@ -35,24 +35,20 @@ class EmojiLine extends StatelessWidget {
...
                     onEmojiSelected(e);
                   },
                   child: Padding(
-                    padding: const EdgeInsets.only(right: 2.0),
-                    child: FittedBox(
-                      child: Text(
-                        e,
-                        style: TextStyle(fontSize: EmojiLine.fontSize),
-                      ),
+                    padding: const EdgeInsets.only(right: 20),
```
[the remark this answers is not in the document]
Comment (ManhNTX) [reply lib/widgets/message/emoji_set.dart:48]: Screenshot 2023-02-10 at 08 05 37
it 24 sorry

**Pull request merged**